DE/Game.po

```diff
@@ -683,14 +683,14 @@ msgstr "Möchten Sie die Rolle wirklich löschen?"
 #: /Game/Blueprint/ACF/Item/Book/AutoCreate/BP_Book_JXX02.Default__BP_Book_JXX02_C.ItemInfo.Name
 msgctxt ",1E0E1014475CBA9F745970B164487715"
 msgid "机修（二卷）"
-msgstr "Maschinenreparatur (Band 2)"
+msgstr "Mechanik (Band Zwei)"
```

Comment:
If you change Maschinenreparatur to Mechanik, please change all Books and the stats